lib/eslint.js

```diff
@@ -830,8 +833,9 @@ module.exports = (function() {
      */
     api.getScope = function() {
         var parents = controller.parents(),
-            innerBlock = null,
-            selectedScopeIndex;
+            scope = currentScopes[0];
+            // innerBlock = null,
+            // selectedScopeIndex;
```

Comment:
Perhaps I missed something - I don't see an explanation for why this leaves commented-out code. Was that intentional, or is this a work in progress?